README.md

```diff
@@ -1,207 +1,47 @@
+Welcome! This is the [Self-Organised](https://we-are-ols.org/openseeds/) (OLS- Self-Organised) program repository. 
+This repository will be used to provide resources for individuals and communities who want to run the OLS Open Seeds program but couldn't get a chance to participate during the earlier cohorts. Also, the resources here could be useful to folks who want to run the program self-space and independently
 
-## About this Repository
 
-This repository operationalises The Turing Way recommendations for structuring project repositories to support reproducibility in data science projects.
-This repository includes files and directories recommended for enabling reproducibility and collaboration in a project, as well as sharing of research objects.
+### Participation Guidelines
 
-This repository follows the recommendations and guidance provided in *[The Turing Way](https://the-turing-way.netlify.app/welcome)* handbook to data science.
-When reusing this repository, please update the information on your README page with information about your project.
+Open Life Science participants adhere to a [code of conduct](CODE_OF_CONDUCT.md). 
+By participating, you are expected to uphold this code. Please report unacceptable behaviour to [team@we-are-ols.org](mailto:team@we-are-ols.org).
 
-### Using this Repository
+### Graduation guide
 
-![screenshot with arrows and instructions describing how to use this repository template to create a project repo. The description provided in the title.](./assets/use-this-repo.png)
+To learn about graduation criteria and process, please visit this [Graduation Guide](https://we-are-ols.org/openseeds/cohort-procedures-and-templates/graduation-guide.html).
 
-**Image Description:**
+For communities, running this program we advise that  you organize graduation as part of the course please consider rehearsing before time  using this template [rehearsal/feedback session on Week 15](https://annuel2.framapad.org/p/ols-9-week-15) and and the graduation using this [graduation calls on Week 16](https://annuel2.framapad.org/p/ols-9-week-15).
 
-Your streamlined project setup is just three steps away:
-* Step 1: Click 'Use this template' on the top right-hand side. Follow it up by clicking 'Create a new repository.'
-* Step 2: In the subsequent window, set the 'owner' (yourself or a GitHub organisation) and give your project repository a descriptive name.
-* Step 3: Finalise by clicking 'Create repository' at the bottom right-hand side.
+Both documents also provide important steps to follow before your graduation.
 
-After creating your own copy of this repository, you can edit or delete existing files and folders, or add new ones as per your needs.
+## Important resources
 
-<!--If reusing this repository, delete this section -->
-## About README
+Please visit notes to find out structure of the programme, notes taken during the session as well as assignments shared at the end of the cohort calls.
```

Comment:
Here you have copied OLS-9 README. You should be creating a version of weekly plans using OLS-9 as reference on this repo and update the table with the right link from this repo.

What is needed is that for each week you are providing resources that an organiser would need to host their weekly sessions.